app/controllers/journey_controller.rb

```diff
@@ -1,11 +1,7 @@
 class JourneyController < ApplicationController
+  before_action :in_game?, only: [:show]
   def show
-    if in_game?
-      render layout: 'wide',  :locals => {:background => params[:slug]}
-    else
-      flash[:error] = "Login and join a game first!"
-      redirect_to root_path
-    end
+    render layout: 'wide',  :locals => {:background => params[:slug]}
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.